### PR TITLE
Implemented Face Player packet

### PR DESCRIPTION
**Pull request by roxxel**

### CODE REVIEW

File: Obsidian/Net/Packets/Play/Clientbound/FacePlayer.cs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        public int Id => 0x37;
+
+        [Field(0), VarLength, ActualType(typeof(int))]
+        public AimType Aim { get; set; }
+        [Field(1)]
+        public double TargetX { get; set; }
+        [Field(2)]
+        public double TargetY { get; set; }
+        [Field(3)]
+        public double TargetZ { get; set; }
```
Comment (Seb-stian) [inline Obsidian/Net/Packets/Play/Clientbound/FacePlayer.cs]: Can be replaced with:
```cs
[Field(1), DataFormat(typeof(double))]
public VectorF Target { get; set; }
```

File: Obsidian/Net/Packets/Play/Clientbound/FacePlayer.cs [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        [Field(5), VarLength, Condition(nameof(IsEntity))]
+        public int EntityId { get; set; }
+
+        [Field(6), VarLength, ActualType(typeof(int)), Condition(nameof(IsEntity))]
+        public AimType AimEntity { get; set; }
+
+        public FacePlayer()
+        {
+
+        }
```
Comment (Seb-stian) [inline Obsidian/Net/Packets/Play/Clientbound/FacePlayer.cs]: Is this needed?

**Review by Seb-stian — COMMENTED** [review]
Thank you for your PR!

File: Obsidian/Net/Packets/Play/Clientbound/FacePlayer.cs (hunk 1 above)
Comment (roxxel) [reply Obsidian/Net/Packets/Play/Clientbound/FacePlayer.cs]: Oh, I forgot that u implemented this

**Review by Seb-stian — APPROVED** [review]
LGTM!

**Comment by Naamloos** [comment]
no hard feelings ♥

**Review by Craftplacer — APPROVED** [review]

**Pull request merged**